crates/opencascade-sys/build.rs

```diff
@@ -52,3 +66,69 @@ fn main() {
     println!("cargo:rerun-if-changed=src/lib.rs");
     println!("cargo:rerun-if-changed=include/wrapper.hxx");
 }
+
+struct OcctConfig {
+    include_dir: std::path::PathBuf,
+    library_dir: std::path::PathBuf,
+    is_dynamic: bool,
+}
+
+impl OcctConfig {
+    /// Find OpenCASCADE library using cmake
+    fn detect() -> Self {
+        // Add path to builtin OCCT
+        #[cfg(feature = "builtin")]
+        {
+            std::env::set_var("DEP_OCCT_ROOT", occt_sys::occt_path().as_os_str());
```

Comment:
Is this environment variable doing anything? I see [documentation](https://cmake.org/cmake/help/latest/envvar/PackageName_ROOT.html) with env vars of the form `<PACKAGE_NAME>_ROOT`, but not `DEP_<PACKAGE_NAME>_ROOT`

Reply:
This is cmake-rs specific thing.
See here [cmake-rs/src/lib.rs#L553](https://github.com/rust-lang/cmake-rs/blob/3ddc3d505ef38ebec54b999110f9dd10b4465252/src/lib.rs#L553)